settlements.civmap.json

```diff
@@ -125,7 +125,7 @@
+    {"name": "State of Ashelor", "x":4320, "z":4857, "nation":"Republic of Adina","contact":"Orinnari /u/Kroolista @Dioxain#2968","web":"https://ashelor.cc","visitors":"welcome","Zoom Visibility":3,"id":"56bcebc6-6d9e-4556-9fc4-7d3491b50e53","collectionId":"62da2fc3-60ba-49a4-898b-817051e47d9d"},
```

Comment:
I consider "State of" and "Republic of" unnecessary clutter on the map, please remove these, if they're really necessary you can add them to the `notes` field

Reply:
Done